tests/code/test_emailHandler.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from flask_mail import Message
+from app.logic.emailHandler import *
+

```

Comment:
We will want to figure out how to test the email sending itself, probably with a combination of Mocking and different configs for testing.

Reply:
testing for the mail has been added. Used ``Flask_Mail``'s testing functionalities.